ADBench/run-all.ps1

```diff
@@ -237,7 +232,7 @@ Class Tool {
 
 					$dir_in = "$([Tool]::gmm_dir_in)$sz/"
 					$dir_out = "$script:tmpdir/gmm/$sz/$($this.name)/"
-					mkdir -force $dir_out
+					New-Item -Path $dir_out -ItemType Directory -ErrorAction Ignore
```

Comment:
Is this equivalent to -Force?  I don't think so, as -Force will squeak if it fails.

Reply:
I'm not sure what you mean.  I don't think we want to be using `-Force` since the documentation sounds rather scary:

> -Force
>
> Force this cmdlet to create an item that writes over an existing read-only item. 

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/new-item?view=powershell-6

I think we just want to ignore errors, i.e. when the directory already exists.

Reply:
Yes, but not when the directory failed to be made, as we will get error cascades.

I think the only thing to do is the "most correct" thing: make a local
```
function mkdir_p($path) {
   if (test-path -type container $path) 
      return
   else if (test-path $path) 
     error "There's something in the way"
   else 
    new-item -itemtype directory $path
}
```

Reply:
OK, done